docs/docs-beta/docs/dagster-plus/access/rbac/users.md

```diff
@@ -1,7 +1,88 @@
 ---
-title: "User management"
+title: "Managing users in Dagster+"
 displayed_sidebar: "dagsterPlus"
-sidebar_position: 1
+sidebar_label: "User management"
+sidebar_position: 10
 ---
 
-# User management in Dagster+
\ No newline at end of file
+Dagster+ allows you to grant specific permissions to your organization's users, ensuring that Dagster users have access only to what they require.
+
+In this guide, you'll learn how to manage users and their permissions using the Dagster+ UI.
+
+<details>
+<summary>Prerequisites</summary>
+
+- A Dagster+ account
+- The required [Dagster+ permissions](/todo):
+   - **Organization Admins** can add, manage, and remove users
+   - **Admins** can add users
+
+</details>
+
+## Before you start
+
+- **If System for Cross-domain Identity Management specification (SCIM) provisioning is enabled,** you'll need to add new users in your identity provider (IdP). Adding users will be disabled in Dagster+.
+- **If using Google for Single sign-on (SSO)**, users must be added in Dagster+ before they can log in.
+- **If using an Identity Provider (IdP) like Okta for SSO**, users must be assigned to the Dagster app in the IdP to be able to log in to Dagster+. Refer to the [SSO setup guides](/todo) for setup instructions for each of our supported IdP solutions.
+
+  By default, users will be granted Viewer permissions on each deployment. The default role can be adjusted by modifying the [`sso_default_role` deployment setting](/todo).
+
+## Adding users
+
+1. Sign in to your Dagster+ account.
+2. Click the **user menu (your icon) > Organization Settings**.
+3. Click the **Users** tab.
+4. Click **Add new user.**
+5. In the **User email** field, enter the user's email address.
+6. Click **Add user**.
+
+After the user is created, you can [add the user to teams](#teams) and [assign user roles for each deployment](#user-roles).
```

Comment:
What happens after a user is added? Do they get an email?

Reply:
Good call - I'll add it in